src/java.base/share/classes/java/nio/charset/exceptions

```diff
@@ -28,7 +28,7 @@
 SINCE=1.4
 PACKAGE=java.nio.charset
 # This year should only change if the generated source is modified.
-COPYRIGHT_YEARS="2000, 2007,"
+COPYRIGHT_YEARS="2000, 2021,"
```

Comment:
Do those modifications to the exception sources end up in a change in `generated` sources?